util/tidy.py

```diff
@@ -0,0 +1,152 @@
+# Copyright 2013 The Servo Project Developers.
```

Comment:
Sorry to be annoying, but I'm not sure I should accept this without knowing who the actual copyright holders are.